src/net.h

```diff
@@ -10,12 +10,14 @@
 #include <addrman.h>
 #include <amount.h>
 #include <bloom.h>
+#include <chainparams.h>
```

Comment:
You can remove this include from net.cpp now that it's included in the header file.

Reply:
Good catch, done